src/components/molecules/HamDropDown/index.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+
+import Styled from 'styled-components';
+import arrowDown from 'assets/images/ham_arrowDown.svg';
+import arrowUp from 'assets/images/ham_arrowUp.svg';
+import Button from '../../atoms/Button/index';
+
+export interface IProps {
+  className?: string;
+  title: string;
+  list: string[];
+}
+
+function HamDropDown({ ...props }: IProps): React.ReactElement {
+  const { title, list } = props;
+  const [opened, setOpened] = useState(false);
+
+  const learnOpenClickListener = (): void => {
+    setOpened(!opened);
+  };
+  const learnDetailClickListener = (e: any): void => {
+    console.log(e.target.value);
+  };
+
+  return (
+    <SHamDropDown>
+      <Button className="title" onClick={learnOpenClickListener}>
+        <>
+          <div className="title__label" style={opened ? { fontWeight: 'bold' } : { fontWeight: 'normal' }}>
+            {title}
+          </div>
+          {opened ? <img className="title__icon" src={arrowUp} /> : <img className="title_icon" src={arrowDown} />}
```

Comment:
```suggestion
          <img className="title__icon" src={opened ? arrowUp : arrowDown} />
```
뒤 컴포넌트의 className에 언더바가 하나 빠졌네요!
그리고 이렇게 코드를 수정하면 조금 더 간단해질 것 같네요